### PR TITLE
Automatically push tagged commits to rubygems

### PR DESCRIPTION
**Pull request by jcarres-mdsol**
Automatic is good
@mdsol/team-10

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 rvm:
-  - 2.0.0
-  - 2.1.10
-  - 2.2.5
-  - 2.3.1
+- 2.0.0
+- 2.1.10
+- 2.2.5
+- 2.3.1
+- 2.4.2
```
Comment (jfeltesse-mdsol) [inline .travis.yml]: - do we really need to support ruby < 2.2 ? 2.0 and 2.1 are EOL.
- kinda weird you have latest 2.4.x while 2.2 and 2.3 are old versions

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,13 @@
 language: ruby
 script: bundle exec cucumber
 rvm:
-  - 2.0.0
-  - 2.1.10
-  - 2.2.5
-  - 2.3.1
+- 2.2.5
+- 2.3.1
+- 2.4.2
```
Comment (ykitamura-mdsol) [inline .travis.yml]: missing indentation?

Comment (ykitamura-mdsol) [reply .travis.yml]: Noticed the comment on [the Grell PR](https://github.com/mdsol/grell/pull/46). 
I see, the Travis tool 👍

**Comment by jfeltesse-mdsol** [comment]
same as comment on grell, any particular reason to keep dated ruby versions? e.g. 2.3.1 and not 2.3.5 for instance.

**Pull request closed without merging**